file_scanning/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM alpine:3.16 as build
+
+# Update & Install Go
+RUN apk update upgrade && apk add --no-cache go
```

Comment:
What was the reason for not using golang alpine `FROM golang:1.19.3-alpine as base`?

Reply:
Well alpine is just under 3mb and golang is over a hundred.. having said that, installing go on alpine makes it even bigger. build time wise they're within a couple of seconds of each other so yes lets switch to golang one for the build container as it's less code :) Good call